modules/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -150,7 +150,7 @@ var newPip = {
 
 var ipConfigurations = concat([
     {
-      name: 'IpConfAzureFirewallSubnet'
+      name: 'pip-${name}-default'
```

Comment:
Out of curiosity: Did it fail because of conflicts?

Reply:
Don't know. After 2h I got the error: "error occured". That change fixed it.

Reply:
I am not sure if that would solve the issue. But I would suggest the name here to not to be the FW name. But the the public IP name in there as well. e.g.: 
`IpConf-pip-adp-carml-az-pip-x-fw` 
and not have the word 'default'

Reply:
Agreed

Reply:
@MrMCake I changed it as discussed, [![Network: AzureFirewalls](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.azurefirewalls.yml/badge.svg?branch=users%2Frahalan%2FUpdateAFW)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.azurefirewalls.yml)